paper/paper.Rmd

```diff
@@ -0,0 +1,308 @@
+
+## Predictions
+
+At a fundamental level, `modelbased` and similar packages leverage model *predictions*.
+These predictions can be of different types, depending on the model and the question at hand.
+For instance, predictions can be associated with **confidence intervals** (`predict = "expectation"`) or **prediction intervals** (`predict = "prediction"`).
+The former corresponds to the uncertainty around the "relationship" (i.e., the conditional estimate, typically of the expectation ($E[X]$) according to a model's parameters)
+while the latter is typically larger and provides information about the range individual observations might take (e.g., _around_ the expectation $E[X]$).
+Moreover, for generalized linear models (GLMs), predictions can be made on the **response scale** (`predict = "response"`) or the **link scale** (`predict = "link"`).
+This corresponds for instance to predictions in terms of probability (response scale) or log odds (link scale) for logistic regression models.
+
+These different types of estimates can be obtained for observations in the original dataset,
+which is useful to assess the model's goodness-of-fit,
+or for new data (typically a "data grid"), which is useful for visualization.
+
+For convenience, the `modelbased` package includes four related functions, that mostly differ in their default arguments for `data` and `predict`:
+
+- `estimate_prediction()`: original data, prediction intervals.
+- `estimate_expectation()`: original data, confidence intervals.
+- `estimate_relation()`: data grid, predictions on the response scale.
+- `estimate_link()`: data grid, predictions on the link scale.
+
+*Note:* if the defaults are changed, then these functions can become redundant.
+For instance, `estimate_relation(..., predict = "link")` will be equivalent to `estimate_link(...)`.
+
+Which function to use depends on the model and the research question at hand (visualizing effects, generating predictions, etc.).
+
+
+## Marginal means and effects
+
+The concept of "marginal" in statistical modeling typically refers to the analysis of the effect of one or more "focus" variables while all other variables are held constant at specific values
```

Comment:
I'm not quite happy with this first paragraph, because it focuses on "effects" (i.e. slopes), while we want to talk about relationship, comparisons/contrasts and effects/slopes. We should maybe focus on the "averaging" aspect (and maybe even move "contrasts" into an own section? - but that's not very necessary)

Reply:
lol you know best here